src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -810,7 +810,7 @@ public string[] GetAvailableProjects()
                     safeAndDisplayName.Add(displayName);
                 }
                 // Sort with respect to the DisplayName
-                var sortedDisplayNames = safeAndDisplayName.OrderBy(i => i.Item2, StringComparer.CurrentCultureIgnoreCase).ToArray();
```

Comment:
I'm reverting this one, because it's working project names, not related to source.